ocs_ci/ocs/ui/base_ui.py

```diff
@@ -62,6 +69,9 @@ def do_click(self, locator, timeout=30):
             logger.error(e)
             raise TimeoutException
 
+    def do_click_by_id(self, id, timeout=30):
+        return self.do_click((id, By.ID), timeout)
```

Comment:
Because some button IDs are defined by variables, such as resource names.
Since it's impossible to dynamically set new locators in runtime, and since `do_click()` only accepts tuples with both a locator and a selector, I found it more elegant to implement the new function that only accepts the locator, instead of passing `(locator, By.ID)` time and time again.
i.e. - 
https://github.com/red-hat-storage/ocs-ci/pull/4391/files#diff-daf5758e27575d1b6d8a43f8d1636e0bcddad9ed31c11010dad2e12973206566R146-R162